### PR TITLE
Fix 3 files

### PR DESCRIPTION
**Pull request by mazocomp**
Please -_-

### CODE REVIEW

File: src/Makefile [hunk 1]
```diff
@@ -89,16 +89,16 @@ ifneq (,$(findstring linux,$(PLATFORM)))
...
 RMFLAGS= -f
 endif
 else
 BIN_SUFFIX=_native
 endif
 endif
-CLIENT_INCLUDES= $(INCLUDES) -I/usr/X11R6/include `sdl2-config --cflags`
-CLIENT_LIBS= -Lenet -lenet -L/usr/X11R6/lib -lX11 `sdl2-config --libs` -lSDL2_image -lSDL2_mixer -lz -lGL
+CLIENT_INCLUDES= $(INCLUDES) `sdl2-config --cflags`
+CLIENT_LIBS= `pkg-config --libs libenet` `sdl2-config --libs` -lSDL2_image -lSDL2_mixer -lz -lGL
```
Comment (graphitemaster) [inline src/Makefile]: enet ships with the game and engine and is built with the game and engine, we don't use system provided enet.

**Comment by graphitemaster** [comment]
So we don't use system provided enet because the version which ships with the game and engine is built with the game and engine for good reason. Mainly system provided enet is not guaranteed to exist as a package on some distributions (it's a relatively small library with not that many users.) And because the author of the library is the same author as the engine and prefers to keep them in synchronization.

**Comment by mazocomp** [comment]
@graphitemaster too bad:
Port:	0ad-0.0.22
Path:	games/0ad/base
Info:	historical real-time strategy game
Maint:	Anthony J. Bentley <anthony@anjbe.name>
Index:	games lang/python
L-deps:	STEM->=1.0.20:net/gloox audio/libvorbis audio/openal converters/libiconv devel/boost devel/libexecinfo devel/nspr devel/sdl2 graphics/png net/curl net/enet net/miniupnp/miniupnpc textproc/icu4c textproc/libxml
B-deps:	archivers/zip devel/gmake lang/python/2.7 shells/bash
R-deps:	STEM-=0.0.22:games/0ad/data devel/desktop-file-utils
Archs:	amd64 i386

Port:	dunelegacy-0.96.4
Path:	games/dunelegacy
Info:	improved opensource version of Dune 2
Maint:	Kirill Bychkov <kirby@openbsd.org>
Index:	games x11
L-deps:	devel/sdl2-mixer net/enet
B-deps:	archivers/bzip2
R-deps:	devel/desktop-file-utils
Archs:	any

Port:	enigma-1.01p7
Path:	games/enigma
Info:	Oxyd/Rock'n'Roll clone
Maint:	The OpenBSD ports mailing-list <ports@openbsd.org>
Index:	games x11
L-deps:	STEM->=1.3.3:net/enet archivers/zipios devel/sdl-image devel/sdl-mixer devel/sdl-ttf graphics/png textproc/xerces-c
B-deps:	devel/gmake
R-deps:	devel/desktop-file-utils x11/gtk+3,-guic
Archs:	any

Port:	redeclipse-1.5.8p0
Path:	games/redeclipse,-main
Info:	redeclipse client
Maint:	Jonathan Gray <jsg@openbsd.org>
Index:	games x11
L-deps:	STEM->=1.3.12:net/enet devel/sdl2-image devel/sdl2-mixer
B-deps:	devel/gmake graphics/GraphicsMagick
R-deps:	STEM->=1.5.8:games/redeclipse,-data devel/desktop-file-utils x11/gtk+3,-guic
Archs:	i386 amd64

Port:	sauerbraten-20130203p2
Path:	games/sauerbraten,-main
Info:	sauerbraten client
Maint:	Jonathan Gray <jsg@openbsd.org>
Index:	games x11
L-deps:	devel/sdl-image devel/sdl-mixer net/enet
B-deps:	archivers/bzip2 devel/gmake
R-deps:	games/sauerbraten,-data
Archs:	i386 amd64

Port:	speeddreams-2.2.1p0
Path:	games/speeddreams,-main
Info:	open source motorsport simulation
Maint:	Pascal Stumpf <pascal@stumpf.co>
Index:	games
L-deps:	audio/libvorbis audio/openal devel/sdl2 graphics/jpeg graphics/openscenegraph graphics/png net/enet
B-deps:	STEM->=1.5.1:devel/ninja archivers/gtar archivers/xz devel/cmake devel/plib
R-deps:	games/speeddreams,-data
Archs:	amd64 i386

Port:	sumwars-0.5.8p4
Path:	games/sumwars
Info:	role playing game
Maint:	Pascal Stumpf <pascal@stumpf.co>
Index:	games lang/lua
L-deps:	audio/freealut audio/libvorbis devel/gettext devel/ois devel/physfs net/enet x11/cegui
B-deps:	STEM->=1.5.1:devel/ninja archivers/bzip2 devel/cmake lang/lua/5.2
R-deps:	lang/lua/5.2
Archs:	amd64 i386

Port:	enet-1.3.13p0
Path:	net/enet
Info:	network library for games
Maint:	Laurent FANIS <laurent.fanis@gmail.com>
Index:	net
L-deps:	
B-deps:	
R-deps:	
Archs:	any

**Comment by mazocomp** [comment]
@graphitemaster you're talking like only RE uses enet, there are plenty of other games which use this lib too.

**Comment by mazocomp** [comment]
@graphitemaster @lsalzman I won't send another PR, so, please, take patch for server browser (it fixes annoying bug: I have to connect to the same server twice). Also I have issues with your Makefile anyway, so, please, take a look at other fixes not associated with enet.

**Comment by mazocomp** [comment]
@graphitemaster @lsalzman Good bye!

**Comment by acerspyro** [comment]
lmao my sides

**Comment by mazocomp** [comment]
@acerspyro I am lazy to make PR, I prefer sending attached diffs in e-mail. I said what change to server browser fixes. Telling what change to Makefile/system-install.mk fixes is not necessary, it is obvious. And excuse me, I do not interact with humans as you like, I am robot, I should kill all humans. So, have a good day sir, I hope these patches won't be accepted.

**Pull request closed without merging**